### DISCUSSION TITLE
1.3.1 Docker image release?

### DISCUSSION THREAD

**Discussion by Tailzip** [php/frankenphp#1169] [General]
Hi,

Looks like latest `1.3.1` wasn't released on Docker Hub due to [failed job in pipeline](https://github.com/dunglas/frankenphp/actions/runs/11841706146/job/32998616971). 🙈 

Thanks!

**Comment by AlliBalliBaba** [comment]
I retriggered the build, the Docker images should now be pushed. fyi @dunglas

**Comment by Tailzip** [comment]
Ok now, thanks!
